data-collector.go

```diff
@@ -17,6 +18,213 @@ import (
+    "id": "/tmp/always-updated.txt",
+    "after": {
+      "owner": "aleff",
+      "group": "wheel",
+      "mode": "0644",
+      "path": "/tmp/always-updated.txt",
+      "verifications": [],
+      "checksum": "f27d6dba15d380b69870a8dc7704f383bc37dd43728d2556b572bbc9e8137b97"
+    },
+    "before": {
+      "checksum": "0f92fd4284f031a9b163bf5207cde0470e522b3d3b6292ced654905a3bdc8f57",
+      "owner": "aleff",
+      "group": "wheel",
+      "mode": "0644",
+      "path": "/tmp/always-updated.txt"
+    },
+    "duration": "8",
+    "delta": "--- /tmp/always-updated.txt\t2016-06-28 09:05:45.000000000 -0400\\n+++ /tmp/.chef-always-updated.txt20160628-74223-6pcmxs\t2016-06-28 11:13:22.000000000 -0400\\n@@ -1,2 +1,2 @@\\n-369adb69-87f6-44a1-b4d5-7aa4fafe45e2\\n+aaa6b8a6-a300-4a7a-a285-f4746ac716fe",
+    "result": "create",
+    "status": "updated",
+    "cookbook_name": "insights-test",
+    "cookbook_version": "0.1.1"
+  },
+  {
+    "type": "file",
+    "name": "/failed/file/resource",
+    "id": "/failed/file/resource",
+    "after": {
+      "owner": null,
+      "group": null,
+      "mode": null,
+      "path": "/failed/file/resource"
+    },
+    "before": {},
+    "duration": "0",
+    "delta": "",
+    "result": "create",
+    "status": "skipped",
+    "cookbook_name": "insights-test",
+    "cookbook_version": "0.1.1",
+    "conditional": "not_if { #code block }"
+  },
+  {
+    "type": "file",
+    "name": "/tmp/do-not-write.txt",
+    "id": "/tmp/do-not-write.txt",
+    "after": {
+      "owner": null,
+      "group": null,
+      "mode": null,
+      "path": "/tmp/do-not-write.txt"
+    },
+    "before": {},
+    "duration": "0",
+    "delta": "",
+    "result": "create",
+    "status": "skipped",
+    "cookbook_name": "insights-test",
+    "cookbook_version": "0.1.1",
+    "conditional": "not_if { #code block }"
+  },
+  {
+    "type": "file",
+    "name": "/path/does/not/exist/but/we/will/ignore/this/failure.txt",
+    "id": "/path/does/not/exist/but/we/will/ignore/this/failure.txt",
+    "after": {
+      "owner": null,
+      "group": null,
+      "mode": null,
+      "path": "/path/does/not/exist/but/we/will/ignore/this/failure.txt"
+    },
+    "before": {},
+    "duration": "0",
+    "delta": "",
+    "result": "create",
+    "status": "skipped",
+    "cookbook_name": "insights-test",
+    "cookbook_version": "0.1.1",
+    "conditional": "not_if { #code block }"
+  },
+  {
+    "type": "file",
+    "name": "/path/does/not/exist/so/this/should/fail.txt",
+    "id": "/path/does/not/exist/so/this/should/fail.txt",
+    "after": {
+      "owner": null,
+      "group": null,
+      "mode": null,
+      "path": "/path/does/not/exist/so/this/should/fail.txt"
+    },
+    "before": {},
+    "duration": "0",
+    "delta": "",
+    "result": "create",
+    "status": "skipped",
+    "cookbook_name": "insights-test",
+    "cookbook_version": "0.1.1",
+    "conditional": "not_if { #code block }"
+  }
+`
```

Comment:
The list's closing bracket `]` is missing. Please add that and indent the resources.